tools/xml2palette/xml2palette.py

```diff
@@ -247,9 +253,17 @@ def process_compounddef(compounddef):
         if len(briefdescription) > 0:
             if briefdescription[0].text is None:
                 print("No brief description text")
-                result.append({"key":"text", "direction":None, "value":""})
+                result.append({
+                    "key":"text",
+                    "direction":None,
+                    "value":""
+                })
             else:
-                result.append({"key":"text", "direction":None, "value":briefdescription[0].text.strip()})
+                result.append({
+                    "key":"text",
+                    "direction":None,
+                    "value":briefdescription[0].text.strip(" .")
```

Comment:
here I added stripping of '.' which doxygen automatically adds for the \brief tag

Reply:
Looks good